### ISSUE TITLE
Bagisto version required

### ISSUE THREAD

**Issue by karthik1987** [devansh-webkul/commerce#19]
This will support Bagisto 2.2.3?

**Comment by amiyoghoshstar** [comment]
So this is dead project?

**Comment by VikasTiwari-Webkul** [comment]
@amiyoghoshstar 
No, We are working on compatibility with latest version of bagisto we will share you an update soon.

**Comment by DzElias** [comment]
A question, what version of bagisto does it work with, I wanted to try it with version 2.2.4 using xampp and ngrok and it doesn't work, I get this error: 
`cause: 'unknown',
  status: 500,
  message: 'Unexpected token \'<\', "<!DOCTYPE "... is not valid JSON',`

**Comment by VikasTiwari-Webkul** [comment]
@DzElias 

Please check with latest release: https://github.com/bagisto/nextjs-commerce/tree/v2.1.0

It is compatible with Bagisto 2.2.3
